precompiles/avs/avs.go

```diff
@@ -84,8 +84,12 @@ func (p Precompile) Run(evm *vm.EVM, contract *vm.Contract, readOnly bool) (bz [
 	// It avoids panics and returns the out of gas error so the EVM can continue gracefully.
 	defer cmn.HandleGasError(ctx, contract, initialGas, &err)()
 
-	if method.Name == MethodAVSAction {
+	switch method.Name {
+	case MethodAVSAction:
 		bz, err = p.AVSInfoRegisterOrDeregister(ctx, evm.Origin, contract, stateDB, method, args)
+	case MethodRegisterOperatorToAVS:
```

Comment:
So won't you support operator deregister from AVS?

Reply:
fixed